doc/index.md

```diff
@@ -1,14 +1,30 @@
 # Introduction
 
-lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. The software supports x86, x64 and Arm based systems, as well as GNU/Linux, OSX, FreeBSD and others [1]. lua-http can be utilized as a server or client and includes the following features: 
+lua-http is an performant, capable HTTP and WebSocket library for Lua. The library supports Lua 5.1 and greater and runs on a wide range of operating systems and platforms. Some of the features of the library include: 
 
-  - HTTP 1 and HTTP 2 as specified by [RFC 7230](https://tools.ietf.org/html/rfc7230) and [RFC 7540](https://tools.ietf.org/html/rfc7540)
+  - HTTP version 1, 1.1 and 2 as specified by [RFC 7230](https://tools.ietf.org/html/rfc7230) and [RFC 7540](https://tools.ietf.org/html/rfc7540)
   - WebSockets as specified by [RFC 6455](https://tools.ietf.org/html/rfc6455) including ping/pong, binary data transfer and TLS encryption
   - Transport Layer Security (TLS) - lua-http supports HTTPS and WSS via [luaossl](https://github.com/wahern/luaossl).
   - Fully asynchronous API that does not block the current thread when executing operations that typically block
+  - Provides both client and server APIs 
   - Easy integration into other event-loop based application models
 
-lua-http was written to fill a gap in the Lua ecosystem by providing an HTTP and WebSocket library with the following traits:
+### Portability
+
+lua-http is pure Lua code with the following external runtime dependancies: 
+
+  - [cqueues](http://25thandclement.com/~william/projects/cqueues.html) - Posix API library for Lua
+  - [luaossl](http://25thandclement.com/~william/projects/luaossl.html) - Lua bindings for TLS/SSL
+  - [openssl](https://www.openssl.org/) - Provides TLS/SSL libraries
```

Comment:
`openssl` is depended on transitively: I'm not sure it should be mentioned explicitly?